src/components/common/Header.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import { Link, useLocation } from 'react-router-dom';
+
+import { menu, close } from 'assets';
+
+import routes from 'routes/routes';
+
+const Header = () => {
+  const Options = [
```

Comment:
Also, if you define the options outside the component, define it as `const OPTIONS = [...`, and if it's inside use camel case convention, in this case `const options = [..`

Reply:
Since routes file returns a react element as object part it needs to be called into a function so i can't define it outside of the header component, it is returning a .tsx file not a plane .ts file.

so we need to use camel convention `options = [...]` inside header component :)